### PR TITLE
Feature/cpt templates

### PR DESCRIPTION
**Pull request by DannyCrews**
Adds a custom page template and archive template with associated styling.

### CODE REVIEW

File: src/single-bulb-learning-module.php [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Template file used to render a single learning module.
+ *
+ * @package bu-learning-blocks
+ */
+
+get_header();
+?>
```
Comment (jdub233) [inline src/single-bulb-learning-module.php]: Closing and opening php tags here also may not be necessary?

File: src/common.scss [hunk 2]
```diff
@@ -14,75 +14,181 @@ $green: #bada55;
...
-
-      &-feedback {
-        font-weight: bold;
-        font-size: .9em;
-      }
-    }
-  }
+.bulb-container {
+	flex: 0 1 auto;
+	// margin-top: 20px;
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/common.scss]: Done

File: src/bulb-learning-module-archive.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+/**
+ * The template for displaying learning module archives
+ *
+ * @package BU Learning Blocks
+ *
+ * @since 0.0.2
+ */
+
+get_header();
+?>
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/bulb-learning-module-archive.php]: Done

File: src/single-bulb-learning-module.php (hunk 1 above)
Comment (DannyCrews) [reply src/single-bulb-learning-module.php]: Done

**Pull request merged**